### ISSUE TITLE
multiple commands fails

### ISSUE THREAD

**Issue by schallee** [paluh/ansible-augeas#31]
I am unable to successfully run multiple commands. I have gone so far as to copy and paste the "Redefine eth0 interface" example and it fails the same way. Running individual commands using "command" instead of "commands" seems to work.

The relevent output from the 'Redefine eth0 interface' example:

```
    "module_stdout": "\r\nTraceback (most recent call last):\r\n  File \"/tmp/ansible_cOmD59/ansible_module_augeas.py\", line 526, in <module>\r\n    main()\r\n  File \"/tmp/ansible_cOmD59/ansible_module_augeas.py\", line 508, in main\r\n    commands = parse_commands(module.params['commands'])\r\n  File \"/tmp/ansible_cOmD59/ansible_module_augeas.py\", line 305, in parse_commands\r\n    value = tokens.__next__()\r\nAttributeError: 'listiterator' object has no attribute '__next__'\r\n", 

```

Running asnible on a debian stretch amd64 with a target host running debian jessie on armhf.

ansible --version on ansible host:

```
ansible 2.4.3.0
  config file = /home/schallee/devel/ansible/ansible.cfg
  configured module search path = [u'/home/schallee/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.13 (default, Nov 24 2017, 17:33:09) [GCC 6.3.0 20170516]

```

python -version on target host:

```

Python 2.7.9
```

dpkg -l | grep augeas:on target host:

```
ii  augeas-lenses                  1.2.0-0.2+deb8u2                        all          Set of lenses needed by libaugeas0 to parse config files
ii  libaugeas0                     1.2.0-0.2+deb8u2                        armhf        Augeas configuration editing library and API
ii  python-augeas                  0.4.1-2                                 all          Python bindings for Augeas

```

Thanks!

**Comment by paluh** [comment]
Hi! Thanks for the report!
I'm not able to test this library till end of next week. The bug was probably introduced by `Python 3` compat PR: https://github.com/paluh/ansible-augeas/pull/29 . I'm not sure how to handle it in both `Python 2` and `Python 3` - could we use `next(tokens)` there?

**Comment by paluh** [comment]
Fixed: 39b8144
